test/test-mda-models/test-data-delivery-spark-model-basic/pom.xml

```diff
@@ -239,5 +236,17 @@
             <version>${version.cucumber.reporting.plugin}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-client</artifactId>
+            <version>${version.test.resteasy}</version>
```

Comment:
I: Upon testing downstream project with spark pipeline I don't see any issue with building and tilitng up.
![Screenshot 2024-10-24 at 10 53 59 AM](https://github.com/user-attachments/assets/6a5d791e-e58a-4a9d-b2e5-469a6b79c573)
![Screenshot 2024-10-24 at 10 54 39 AM](https://github.com/user-attachments/assets/0b5e68b7-46b3-4839-955e-43cc4d073cc6)

Reply:
To follow up on this, it appears this is an issue with the `foundation-alerting` import.  There is likely some version incompatibility that was caused by the Java 17 upgrades.  It's worth noting that it seems there is a bug where the mda generation always things alerting is disabled even when it isn't.  We will likely need to handle both of these issues before releasing 1.10.